benchexec/tablegenerator/react-table/src/utils/utils.js

```diff
@@ -661,35 +691,27 @@ const makeFilterDeserializer =
 
 const makeUrlFilterSerializer = (statusValues, categoryValues) => {
   const serializer = makeFilterSerializer({ statusValues, categoryValues });
-  return (filter, options) => {
-    const previousParams = getHashSearch();
+  return (filter, history) => {
     if (!filter) {
-      return setHashSearch(previousParams, options);
+      return setParam({}, history);
```

Comment:
I think this should also delete the `filter` like the case below.

Reply:
Okay!